account/src/main/java/com/user/account/entity/Transactions.java

```diff
@@ -0,0 +1,50 @@
+package com.user.account.entity;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table (name ="transactions" )
+public class Transactions {
+	
+	@Id
+	private long transactionID;
+	private long accountNumber;
+	private long amount;
+	private String transactionType;
+	private static long maxValue = 1;
+	
+	//Constructor
+	public Transactions ( long accountNumber, long amount, String transactionType){
+		this.transactionID = maxValue;
+		maxValue++;
+		this.accountNumber = accountNumber;
+		this.amount = amount;
+		this.transactionType = transactionType;
+	}
+	
+	public Transactions() {
+		

```

Comment:
Seems unnecessary

Reply:
Getting error because DAO layer was not able to insert into the table without default constructor

Reply:
Prefer `@NoArgsConstructor`

Reply:
Yeah Nitin Already added in later Commits